### ISSUE TITLE
Running .update() fails (column "updatedAt" of relation "posts" does not exist)

### ISSUE THREAD

**Issue by hidjou** [eveningkid/denodb#143]
Trying to update a DB record and it fails because it's trying to set the value of 'updatedAt' on the DB table instead of 'updated_at'

### Error

```
PostgresError: column "updatedAt" of relation "posts" does not exist
    at parseError (error.ts:106:10)
    at Connection._processError (connection.ts:434:19)
    at Connection._simpleQuery (connection.ts:298:20)
    at async Connection.query (connection.ts:546:16)
    at async Client.query (client.ts:25:12)
    at async PostgresConnector.query (postgres-connector.ts:60:22)
    at async Database.query (database.ts:159:21)
    at async posts.ts:51:5
    at async dispatch (middleware.ts:41:7)
    at async dispatch (middleware.ts:41:7)
```

### Code

```
...
const post: Post = await Post.where('uuid', params.uuid).first()    // works
post.body = body
await post.update() // errors
...
```

What's strange is that if I change `updatedAt` to `updated_at` on line 453 in the model source file in the library files my update call runs just fine

```
if (this.timestamps) {
    fieldsToUpdate[this.formatFieldToDatabase("updated_at") as string] = new Date();
}
```

Obviously I dont want to rely on changing the source code of the library to get it to work, am I not configuring something properly ?

(below is my Post model)

```
import { DataTypes, Model } from 'https://deno.land/x/denodb@v1.0.12/mod.ts'
import { v4 } from 'https://deno.land/std@0.74.0/uuid/mod.ts'

class Post extends Model {
  static table = 'posts'

  static timestamps = true

  static fields = {
    id: { primaryKey: true, autoIncrement: true },
    // NOTE: This assumes not null
    uuid: { type: DataTypes.UUID },
    username: { type: DataTypes.STRING },
    body: { type: DataTypes.STRING },
  }

  body!: string
  username!: string

  static defaults = {
    uuid: v4.generate(),
  }
}

export default Post
```

Please let me know if you have a fix for this, thanks a lot!

**Comment by juslin03** [comment]
Hello, How to contribute to the project ?

**Comment by eveningkid** [comment]
Hi Ahmed,

Long wait I know, I only got time to get back to this project. Hope you don't mind!

If you are still around, I figured out the issue was laying there:

https://github.com/eveningkid/denodb/blob/b98aac4bd331ba145467a825d879c7cc80d6002d/lib/translators/sql-translator.ts#L222-L227

In the SQL translator, it was using `camelCase` instead of `snake_case` when translating a property to its database counterpart!

So you would get `updatedAt => updatedAt` instead of `updated_at`.

This is only relevant for SQL databases of course so I will just change that file.
